spec/diffux_ci_spec.rb

```diff
@@ -523,4 +523,65 @@ def snapshot_file_exists?(description, size, file_name)
         .to eq(false)
     end
   end
+
+  describe 'when additional files are served from public directories' do
+    before do
+      @tmp_pub_dir = File.join(@tmp_dir, 'public')
+      Dir.mkdir(@tmp_pub_dir)
+
+      File.open(File.join(@tmp_pub_dir, 'picture.gif'), 'wb') do |f|
+        f.write(Base64.decode64('R0lGODlhAQABAIABAP///wAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=='))
```

Comment:
It would be nice to move this base64 encoded string into a well named variable to make this code more readable.